src/user/managing-dashboards.md

```diff
@@ -231,28 +182,40 @@ A dialog opens where the filter selection can be made.
-are lost.
-Filter badges appear above the dashboard items to
-indicate that what is shown in the dashboard items is not the original
-visualization, but a manipulated one where the filters override
-the stored dimensions' values.
+Filters are not stored, so when switching to a different dashboard they are lost. Black filter badges appear above the dashboard items to indicate that what is shown in the dashboard items is not the original visualization, but a filtered one where the filters override the stored dimensions' values.
 
 ![Current filters displayed as badges above the dashboard](resources/images/dashboard/dashboard-filter-badges.png)
 
-Filter badges can be clicked for opening the filter selection dialogs
-thus allowing for filter editing.
-A filter can be removed by clicking on the **Remove** button in the badge.
-Whenever a filter is added, edited or removed, the dashboard items
-reload to show the updated data.
-Filter badges are always visible at the top of the page when
-scrolling the dashboard content.
+Filter badges can be clicked for opening the filter selection dialogs thus allowing for filter editing. A filter can be removed by clicking on the **Remove** button in the badge. Whenever a filter is added, edited or removed, the dashboard items reload to show the updated data. The filter badges are always visible at the top of the page when scrolling the dashboard content.
 
 By default, users are able to filter dashboard items by any dimension defined in your system. To limit available filters, see [Restricting dashboard filters](#restricting-dashboard-filters).
 
+### Making dashboards available offline
+
+To make a dashboard available offline, choose the **Make dashboard available offline** option in the **...More** menu. This will cause a reload of the dashboard where requests to the server are recorded and saved in browser storage. Note that offline dashboards are only available on the computer and browser where you set it to offline. If you currently have a filter applied when requesting the dashboard be made available offline, the filter will have to be removed first.
+
+Dashboards that have been saved for offline have an indicator on the dashboard chip in the dashboards bar, as well as a tag showing the time it was saved.
+
+![](resources/images/dashboard/dashboard-offline-dashboard.png)
+
+If the dashboard has been changed since you made it available offline, either by you or someone else, you'll need to choose **Sync offline data now** from the **...More** menu to save the latest version of the dashboard.
+
+
+![](resources/images/dashboard/dashboard-sync-offline.png)
+
+You can remove a dashboard from offline storaged by choosing **Remove from offline storage** in the **...More** menu.
+
+In addition, if you manually log out of the DHIS2 instance, all cached dashboards will be cleared and they will no longer be available offline.
+
+
+![](resources/images/dashboard/dashboard-remove-offline.png)
```

Comment:
This screenshot should be moved up slightly to be next to the "You can remove a dashboard from offline...." section

Reply:
done